### PR TITLE
Add German translation

### PR DESCRIPTION
**Pull request by gregorni**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -0,0 +1,628 @@
...
+#. One name per line, please do not remove previous names.
+#: paleta/app.py:52
+msgid "translator-credits"
+msgstr ""
+"gregorni https://gitlab.com/gregorni\n"
+
+#. Translators: used in the phrase "Powered by color-thief"
+#: paleta/app.py:55
+msgid "Powered by "
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:29]: Missing translation

File: po/de.po [hunk 2]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/window.py:64
+msgid "Unable to load database."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:33]: Missing translation

File: po/de.po [hunk 3]
```diff
@@ -0,0 +1,628 @@
...
+msgstr "Bild öffnen"
+
+#: paleta/window.py:132
+msgid "Image files"
+msgstr "Bilddateien"
+
+#. Translators: Do not replace {}
+#: paleta/window.py:163
+msgid "Could not open image: {}"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:51]: Missing translation

File: po/de.po [hunk 4]
```diff
@@ -0,0 +1,628 @@
...
+
+#. Translators: Do not replace {base_name}, {SUCCESS_GREEN}, or the span tags, only translate "Opened image:"
+#: paleta/window.py:168
+#, python-brace-format
+msgid "<span foreground={SUCCESS_GREEN}>Opened image:</span>  {base_name}"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:57]: Missing translation

File: po/de.po [hunk 5]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/ui/help-overlay.ui:47
+msgctxt "shortcut window"
+msgid "Save Extracted Palette"
+msgstr "Extrahierte Palette speichern"
+
+#: paleta/ui/help-overlay.ui:53
+msgctxt "shortcut window"
+msgid "Go To Image Drop Page"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:128]: Missing translation

File: po/de.po [hunk 6]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/ui/help-overlay.ui:72
+msgctxt "shortcut window"
+msgid "Add Palette"
+msgstr "Palette hinzufügen"
+
+#: paleta/ui/help-overlay.ui:78
+msgctxt "shortcut window"
+msgid "Go To Palette Page"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:148]: Missing translation

File: po/de.po [hunk 7]
```diff
@@ -0,0 +1,628 @@
...
+msgstr "_Über Paleta"
+
+#: paleta/dialog/add_color_dialog.py:29
+msgid "Add Color to {}"
+msgstr "Farbe zu {} hinzufügen"
+
+#. Translators: Do not replace {}
+#: paleta/dialog/add_color_dialog.py:31 paleta/dialog/add_color_dialog.py:55
+msgid "Choose new color to add to {}"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:177]: Missing translation

File: po/de.po [hunk 8]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/dialog/add_new_palette_dialog.py:32
+#: paleta/dialog/add_new_palette_dialog.py:58
+msgid "Choose color to add to new palette"
+msgstr ""
+
+#: paleta/dialog/add_new_palette_dialog.py:42
+msgid "Pick a color to add to new palette."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:211]: Missing translation

File: po/de.po [hunk 9]
```diff
@@ -0,0 +1,628 @@
...
+
+#. Translators: Do not replace {color.hex}
+#: paleta/dialog/add_new_palette_dialog.py:51
+#, python-brace-format
+msgid "Currently selected color: {color.hex}"
+msgstr "Ausgewählte Farbe: {color.hex}"
+
+#: paleta/dialog/add_new_palette_dialog.py:83
+msgid "Unable to add palette, must select a color."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:221]: Missing translation

File: po/de.po [hunk 10]
```diff
@@ -0,0 +1,628 @@
...
+#. Translators: Do not replace {name}
+#: paleta/dialog/add_new_palette_dialog.py:95
+#: paleta/dialog/dialog_windows.py:41
+#, python-brace-format
+msgid "Unable to add new palette «{name}»"
+msgstr "Neue Palette «{name}» konnte nicht hinzugefügt werden"
+
+#: paleta/dialog/delete_color_dialog.py:37
+msgid "Remove color {} from {}?"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:243]: Missing translation

File: po/de.po [hunk 11]
```diff
@@ -0,0 +1,628 @@
...
+
+#. Translators: Do not replace {self.name}
+#: paleta/dialog/dialog_windows.py:81
+#, python-brace-format
+msgid "Unable to rename palette «{self.name}»."
+msgstr "Palette «{self.name} konnte nicht umbenannt werden."
+
+#: paleta/dialog/dialog_windows.py:90
+msgid "Palette"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:284]: Missing translation

File: po/de.po [hunk 12]
```diff
@@ -0,0 +1,628 @@
...
+msgid "_Cancel"
+msgstr "_Abbrechen"
+
+#: paleta/dialog/ui/add_color_dialog.ui:15
+msgid "_Add Color"
+msgstr "Farbe _hinzufügen"
+
+#: paleta/dialog/ui/add_color_dialog.ui:41
+msgid "Select from previously added colors or pick a new one."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:356]: Missing translation

File: po/de.po [hunk 13]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/dialog/ui/add_color_dialog.ui:63
+#: paleta/dialog/ui/add_new_palette_dialog.ui:76
+msgid "_Choose with Picker"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:361]: Missing translation

File: po/de.po [hunk 14]
```diff
@@ -0,0 +1,628 @@
...
+msgstr "Neue Palette hinzufügen"
+
+#: paleta/dialog/ui/add_new_palette_dialog.ui:15
+msgid "_Add Palette"
+msgstr "Palette _hinzufügen"
+
+#: paleta/dialog/ui/add_new_palette_dialog.ui:43
+#: paleta/dialog/ui/rename_palette_dialog.ui:24
+msgid "New Palette _Name"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:375]: Missing translation

File: po/de.po [hunk 15]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/dialog/ui/add_new_palette_dialog.ui:53
+msgid "Select from previously added colors or pick one to add to new palette."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:379]: Missing translation

File: po/de.po [hunk 16]
```diff
@@ -0,0 +1,628 @@
...
+msgid "Unable to start palette extraction, no image loaded."
+msgstr ""
+
+#: paleta/pages/image_drop/color_thief_panel.py:117
+msgid "Unable to extract colors from image."
+msgstr "Farben konnten nicht aus dem Bild extrahiert werden."
+
+#: paleta/pages/image_drop/color_thief_panel.py:128
+msgid "Unable to save palette, no image loaded."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:447]: Missing translation

File: po/de.po [hunk 17]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/color_thief_panel.py:133
+msgid "Unable to save palette, no colors extracted."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:451]: Missing translation

File: po/de.po [hunk 18]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/image_drop_page.py:52
+#: paleta/pages/image_drop/image_drop_page.py:67
+msgid "Unable to read drop."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:456]: Missing translation

File: po/de.po [hunk 19]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/image_drop_page.py:57
+msgid ""
+"Unable to verify file on drop, try with the file chooser in the upper left "
+"corner."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:462]: Missing translation

File: po/de.po [hunk 20]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/ui/color_thief_panel.ui:63
+msgid "Palette Count"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:466]: Missing translation

File: po/de.po [hunk 21]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/ui/color_thief_panel.ui:64
+msgid "Max amount of colors to retrieve."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:470]: Missing translation

File: po/de.po [hunk 22]
```diff
@@ -0,0 +1,628 @@
...
+msgid "Low"
+msgstr "Niedrig"
+
+#: paleta/pages/image_drop/ui/color_thief_panel.ui:155
+msgid "_Save Palette"
+msgstr "Palette _speichern"
+
+#: paleta/pages/image_drop/ui/image_drop_page.ui:14
+msgid "Drop Image"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:494]: Missing translation

File: po/de.po [hunk 23]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/image_drop/ui/image_drop_page.ui:15
+msgid ""
+"Drag an image here, click the button below, or click the button in the upper "
+"left corner of this window."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:500]: Missing translation

File: po/de.po [hunk 24]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/palette/palette_page.py:61
+msgid "Cannot toggle edit mode, no palettes added."
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:504]: Missing translation

File: po/de.po [hunk 25]
```diff
@@ -0,0 +1,628 @@
...
+
+#: paleta/pages/palette/ui/palette_page.ui:43
+msgid "Add Some Palettes!"
+msgstr ""
+
+#: paleta/pages/palette/ui/palette_page.ui:44
+msgid ""
+"Drop an image onto the drop page and extract some colors, or add a palette "
+"manually by clicking the button in the lower right corner to get started!"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:526]: Missing translation

File: po/de.po [hunk 26]
```diff
@@ -0,0 +1,628 @@
...
+msgid "Duplicate Palette"
+msgstr "Palette duplizieren"
+
+#: paleta/pages/palette/ui/palette_row.ui:80
+msgid "Delete Palette"
+msgstr "Palette löschen"
+
+#: paleta/pages/palette/ui/palette_page.ui:43
+msgid "Add Some Palettes!"
+msgstr ""
```
Comment (nate-xyz) [inline po/de.po:520]: Missing translation

**Review by nate-xyz — CHANGES_REQUESTED** [review]

**Comment by gregorni** [comment]
Oops, sorry, I uploaded the wrong de.po, I had several copies. Give me a second.

**Comment by gregorni** [comment]
It's fixed now. I'm sorry for making you that trouble, I should've noticed the missing strings.

**Comment by gregorni** [comment]
Btw, next time maybe don't go through the effort of marking every single missing string, just say that a lot of strings are empty, the translator ought to look for them themselves.

**Comment by nate-xyz** [comment]
Good point lol

**Comment by nate-xyz** [comment]
Thanks!

**Pull request merged**